src/D2L.Security.OAuth2/Keys/Development/InMemoryPublicKeyDataProvider.cs

```diff
@@ -12,6 +13,10 @@ namespace D2L.Security.OAuth2.Keys.Development {
 	/// </summary>
 	[Obsolete("Only use this in tests and for prototyping without a db")]
 	public sealed class InMemoryPublicKeyDataProvider : IPublicKeyDataProvider {
+		[Mutability.Audited(
+			"Todd Lang",
+			"14-Aug-2018",
+			"Only used in tests, mutability not a concern here.")]

```

Comment:
If this is only used in tests, why not move it to a project that uses the less strict TestAnalyzer?  I feel like we already ignore test-only things in other assemblies without resorting to `Mutability.Audited`